unittests/test_policies.py

```diff
@@ -104,10 +107,28 @@ def num_failures_stage(runner, stage):
     return len([t for t in stats.failures() if t.failed_stage == stage])
 
 
+def open_json_schema():
+    # Open and store the JSON schema for later validation
+    schema_filename = os.path.join(reframe.INSTALL_PREFIX, 'reframe',
+                                   'schemas', 'report.json')
```

Comment:
Unit tests are meant to run from the top-level directory, so the following should be enough:

```suggestion
    schema_filename = 'reframe/schemas/runreport.json'
```